ipywidgets/widgets/widget.py

```diff
@@ -483,7 +483,7 @@ def _compare(self, a, b):
         else:
             return a == b
 
-    def _compare_buffers(self, a, b, key):
+    def _buffer_list_equal(self, a, b):
```

Comment:
Actually, this comparison depends on the top-level put/remove buffer functions, so I factored out the comparison to also be a top-level function.